### PR TITLE
Updates to latest colors to be in compliance with WPiOS/Calypso design

### LINKED ISSUE #330: Update colors to use standard colors
Related to the Gridicons update https://github.com/wordpress-mobile/WordPressCom-Stats-iOS/issues/327, colors (of text and icons) need to be updated to use standard colors:

Essentially, colors should be:

- Text Links - `wordPressBlue`
- Text (non-links) - `darkGrey`
- Icons (for Views, Visitors, Likes, Comments) and accompanying text - `darkGrey`
- Other icons - `wordPressBlue`
- Module column heading - `grey`

Equivalent hex values:

- `wordPressBlue` = `#0087be`
- `grey` = `#87a6bc`
- `darkGrey` = `#2e4453`

/cc @daniloercoli

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #330, #325 

1. Updates colors as requested.
1. Made Insights Latest Post Summary likes & comments not tappable.
1. Made the Insights Latest Post Summary post title tappable.
1. Fixed View Web Stats to load properly - URL apparently changed on WordPress.com.
1. Fixed View All not working right in Insights sections.
1. Removed some dead code in Insights.

Needs Review: @jancavan (design), @jleandroperez (code)

Updated screenshots:
simulator screen shot nov 6 2015 12 37 38 pm
simulator screen shot nov 6 2015 12 37 43 pm
simulator screen shot nov 6 2015 12 37 48 pm
simulator screen shot nov 6 2015 12 37 51 pm
simulator screen shot nov 6 2015 12 37 57 pm
simulator screen shot nov 6 2015 12 39 14 pm
simulator screen shot nov 6 2015 12 39 25 pm

### CODE REVIEW

**Comment by jancavan** [comment]
@astralbodies other than the links, items that expand need the text to be in `wordPressBlue` as well. Sorry if this wasn't clear:

**Tags:**


**Period:**

(We don't currently do this on the web version, but then again we have a different kind of layout on there.)



**Authors** (and Author's posts when expanded) if you haven't done this already.

Other than these, all the other color updates look good!

File: WordPressCom-Stats-iOS/UI/InsightsTableViewController.m [hunk 1]
```diff
@@ -281,12 +281,13 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
...
     StatsSection statsSection = [self statsSectionForTableViewSection:indexPath.section];
+    NSString *identifier = [self cellIdentifierForIndexPath:indexPath];
     
-    if ([[self cellIdentifierForIndexPath:indexPath] isEqualToString:StatsTableViewAllCellIdentifier] ||
+    if ([identifier isEqualToString:StatsTableViewAllCellIdentifier] ||
         (statsSection == StatsSectionInsightsTodaysStats) ||
         (statsSection == StatsSectionInsightsLatestPostSummary)) {
         return indexPath;
-    } else if ([[self cellIdentifierForIndexPath:indexPath] isEqualToString:StatsTableTwoColumnCellIdentifier]) {
+    } else if ([identifier isEqualToString:StatsTableTwoColumnCellIdentifier]) {
```
Comment (jleandroperez) [inline WordPressCom-Stats-iOS/UI/InsightsTableViewController.m]: Nice touch :smile:

**Comment by jleandroperez** [comment]
Code is looking great!. 

Please, let me know whenever @jancavan's comments are addressed, so that we can take it for another ride.

Thanks!

**Comment by astralbodies** [comment]
@jancavan - updated the color logic for those rows - 

simulator screen shot nov 10 2015 1 24 32 pm
simulator screen shot nov 10 2015 1 24 48 pm

**Comment by astralbodies** [comment]
Thanks for the reviews @jancavan and @jleandroperez!

**Pull request merged**